frontend/test/metabase/lib/expressions/resolver.unit.spec.js

```diff
@@ -96,7 +96,7 @@ describe("metabase/lib/expressions/resolve", () => {
       expect(() => filter(["<=", ["lower", A], "P"])).not.toThrow();
     });
 
-    it("should reject a less/greater comparison on functions returning boolean", () => {
+    it.skip("should reject a less/greater comparison on functions returning boolean", () => {
```

Comment:
Why did you skip the spec? Seems like desired behavior, correct me if I'm wrong

Reply:
Reenabled